### PR TITLE
Add return type to compiled macro

### PR DESCRIPTION
**Pull request by ruudk**
This makes it easier for TwigStan to analyze the return type.

### CODE REVIEW

**Comment by ruudk** [comment]
/cc @fabpot @stof 

This would be good to support macros in TwigStan:
- https://github.com/twigstan/twigstan/issues/4

File: src/Node/MacroNode.php [hunk 1]
```diff
@@ -76,7 +77,7 @@ public function compile(Compiler $compiler): void
 
         $compiler
             ->raw('...$varargs')
-            ->raw(")\n")
+            ->raw("): string | Markup\n")
```
Comment (fabpot) [inline src/Node/MacroNode.php]:
```suggestion
            ->raw("): string|Markup\n")
```

**Comment by fabpot** [comment]
I don't think we should merge it yet. I'd like for macros to yield instead.

**Comment by ruudk** [comment]
Since it's compiled code, we change it to become iterable later?

File: src/Node/MacroNode.php (hunk 1 above)
Comment (ruudk) [reply src/Node/MacroNode.php]: Fixed.

**Comment by fabpot** [comment]
Thank you @ruudk.

**Pull request merged**